src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/CreateCaseTest.java

```diff
@@ -162,26 +145,25 @@ private AboutToStartOrSubmitCallbackResponse invokeCallbackEndpoint(
             .header(SyntheticHeaders.SYNTHETIC_TEST_SOURCE, "Bulk Scan Orchestrator Functional test")
             .header(AUTHORIZATION, ccdAuthenticator.getUserToken())
             .header(CcdCallbackController.USER_ID, ccdAuthenticator.getUserDetails().getId())
-            .body(callbackRequest)
+            .body(
+                CallbackRequest
+                    .builder()
+                    .eventId(EventIds.CREATE_NEW_CASE)
+                    .caseDetails(exceptionRecord.toBuilder().build())
```

Comment:
why do we need `toBuilder().build()`?

Reply:
not sure. the (now gone) variable had a mysterious name `exceptionRecordWithSearchFields`

Reply:
just checked case details. the only difference is just creating brand new object - no hidden changes

Reply:
ok, removing